### ISSUE TITLE
Optimization script AttributeError: not configured for detailed-electricity-pricing 

### ISSUE THREAD

**Issue by Jack-Hawthorne** [architecture-building-systems/CityEnergyAnalyst#1883]
I am trying to run a stripped down version of the optimization script for documentation purposes. See below: 
```
Running `cea optimization` with the following parameters:
- general:scenario = c:\reference-case-open\WTP_CBD_h
  (default: C:\reference-case-open\baseline)
- general:region = SG
  (default: CH)
- general:weather = c:\users\jack\documents\github\cityenergyanalyst\cea\databases\weather\Singapore.epw
  (default: c:\users\jack\documents\github\cityenergyanalyst\cea\databases\weather\Zug.epw)
- general:district-heating-network = False
  (default: False)
- general:district-cooling-network = True
  (default: False)
- thermal-network:network-type = DC
  (default: DH)
- thermal-network:network-names = []
  (default: [])
- optimization:initialind = 3
  (default: 3)
- optimization:halloffame = 5
  (default: 20)
- optimization:ngen = 2
  (default: 3)
- optimization:fcheckpoint = 1
  (default: 1)
- optimization:maxtime = 7000
  (default: 604800)
- optimization:recoverycheckpoint = 0
  (default: 0)
- optimization:random-seed = None
  (default: None)
- decentralized:ahuflag = True
  (default: False)
- decentralized:aruflag = True
  (default: False)
- decentralized:scuflag = True
  (default: False)
- decentralized:ahuaruflag = True
  (default: False)
- decentralized:ahuscuflag = True
  (default: False)
- decentralized:aruscuflag = True
  (default: False)
C:\Users\Jack\Miniconda2\envs\cea\lib\site-packages\deap\tools\_hypervolume\pyhv.py:33: ImportWarning: Falling back to the python version of hypervolume module. Expect this to be very slow.
  "module. Expect this to be very slow.", ImportWarning)
C:\Users\Jack\Miniconda2\envs\cea\lib\site-packages\scipy\optimize\_minimize.py:37: ImportWarning: Not importing directory 'C:\Users\Jack\Miniconda2\envs\cea\lib\site-packages\scipy\optimize\lbfgsb': missing __init__.py
  from .lbfgsb import _minimize_lbfgsb
3
Traceback (most recent call last):
  File "C:\Users\Jack\Miniconda2\envs\cea\Scripts\cea-script.py", line 11, in <module>
    load_entry_point('cityenergyanalyst', 'console_scripts', 'cea')()
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\interfaces\cli\cli.py", line 65, in main
    script_module.main(config)
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\tests\trace_inputlocator.py", line 53, in main
    script_func(config)  # <------------------------------ this is where we run the script!
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\api.py", line 26, in script_runner
    script_module.main(config)
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\optimization\optimization_main.py", line 134, in main
    moo_optimization(locator=locator, weather_file=weather_file, gv=gv, config=config)
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\optimization\optimization_main.py", line 62, in moo_optimization
    lca = LcaCalculations(locator, config.region, config.detailed_electricity_pricing)
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\config.py", line 47, in __getattr__
    return getattr(self.sections['general'], cid)
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\config.py", line 236, in __getattr__
    "Parameter not configured to work with this script: {%s}" % self.parameters[cid].fqname)
AttributeError: Parameter not configured to work with this script: {general:detailed-electricity-pricing}
```
Looks as though it arises when moo_optimization is attempting to use the detailed-electricity-pricing parameter as an argument for LcaCalculations

**Comment by Jack-Hawthorne** [comment]
@daren-thomas you were working on these LcaCalculation invocations a few weeks ago, is it an easy fix? Similar to decentralized, I cannot run the trace for optimization without it working for either the baseline or WTP_CBD_h scenarios.

**Comment by Jack-Hawthorne** [comment]
this looks to be resolved with pr #1878

**Comment by Jack-Hawthorne** [comment]
closing issue
